docs/README.md

```diff
@@ -0,0 +1,219 @@
+GoReleaser builds Go binaries for several platforms, creates a GitHub release and then
+pushes a Homebrew formula to a repository. All that wrapped in your favorite CI.
+
+[![Release](https://img.shields.io/github/release/goreleaser/goreleaser.svg?style=flat-square)](https://github.com/goreleaser/goreleaser/releases/latest)
+[![Software License](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square)](LICENSE.md)
+[![Travis](https://img.shields.io/travis/goreleaser/goreleaser.svg?style=flat-square)](https://travis-ci.org/goreleaser/goreleaser)
+[![Go Report Card](https://goreportcard.com/badge/github.com/goreleaser/goreleaser?style=flat-square)](https://goreportcard.com/report/github.com/goreleaser/goreleaser)
+[![Powered By: GoReleaser](https://img.shields.io/badge/powered%20by-goreleaser-green.svg?style=flat-square)](https://github.com/goreleaser)
+
+This project adheres to the Contributor Covenant [code of conduct](https://github.com/goreleaser/goreleaser/blob/master/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.   
+We appreciate your contribution. Please refer to our [contributing guidelines](/CONTRIBUTING.md).
+
+# Documentation
+
+- [Introduction](#intorduction)
+- [Quick start](#quick-start)
+- [Environment setup](#environment-setup)
+- [Release customization](#release-customization)
+- [Integration with CI](#integration-with-ci)
+
+##  Introduction
+
+GoReleaser is a release automation tool for Golang projects, the goal is to simplify the build, release and publish steps while providing variant customization options for all steps.  
+
+GoReleaser is built for CI tools, you only need a [single command line](#integration-with-ci) in your build script. Therefore, no package is required.  
```

Comment:
That comma should be a semicolon: `GoReleaser is built for CI tools; you only need`